### PR TITLE
Exclude more python tests in the FVTR

### PR DESCRIPTION
**Pull request by er-1**
In order for the FVTR to work, we need reliable tests.
Unfortunately, some of the python tests have been failing intermittently,
making it difficult to trust the entire python FVTR test.

In order to improve this situation, this patch excludes the following
python tests from the FVTR:
 - test_asyncgen
 - test_logging
 - test_poplib
 - test_threading

Signed-off-by: Erwan Prioul <erwan@linux.ibm.com>

### CODE REVIEW

**Comment by tuliom** [comment]
`AT 10.0 Ubuntu16 ppc64le 697` failed during deb package creation with:

```
rsync -a --delete --delete-excluded \
      --exclude=etc/ld.so.cache \
      --exclude=compat/include \
      --exclude=share/info/dir \
      /home/tcbot/bot-slave/at_opt/at10.0 debian/tmp//home/tcbot/bot-slave/at_opt/at10.0/../
semop(1): encountered an error: Invalid argument
debian/rules:55: recipe for target 'binary-cp' failed
libfakeroot, when sending message: Invalid argument
make[1]: *** [binary-cp] Error 1
```

This is an unexpected error.  Let me try again...

**Comment by er-1** [comment]
> I'd like to wait for the results on AT 10.0 Ubuntu 16.04.

The build #701 passed \o/
